lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -36,6 +38,11 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
         for chapter in response.data['courseware_summary']:
             self.assertIsNotNone(chapter)
         self.assertIn('settings/grading/' + str(self.course.id), response.data['studio_url'])
+        self.assertEqual(response.data['credit_support_url'], CREDIT_SUPPORT_URL)
+        self.assertIsNotNone(response.data['verification_data'])
+        if enrollment_mode == CourseMode.AUDIT:
+            self.assertEqual(response.data['verification_data']['status'], 'none')
+        self.assertNotEqual(response.data['verification_data']['status'], '')
```

Comment:
What is the value of `'status'` in this case? Can we assert something better than not empty string?